commands/commands.go

```diff
@@ -9,9 +9,11 @@ import (
 	"syscall"
 	"text/tabwriter"
 
+	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 
 	"github.com/buildpack/pack"
+	env "github.com/buildpack/pack/internal"
```

Comment:
This import alias is a little awkward. I think you meant to make an env package in internal?